backend/src/main/java/ca/bc/gov/app/controller/client/ClientSubmissionController.java

```diff
@@ -158,8 +157,21 @@ public Mono<Void> submitStaff(
         request.businessInformation().businessName()
     );
 
-    return
-        validator.validate(request, ValidationSourceEnum.STAFF)
+    return Mono.just(
```

Comment:
You can make this block non-reactive by using an if and returning a `Mono.error` instead of this entire block.
Also, the `switchIfEmpty` will never kick in as you're building the object yourself and not using `Mono.justOrEmpty` and in case of null objects, it will instead throw an exception.